intelmq/tests/bots/parsers/generic/test_parser_csv_invalid_values.py

```diff
@@ -45,13 +49,13 @@ def set_bot(cls):
                          "type": "malware",
                          "default_url_protocol": "http://"}
 
-    def test_error(self):
-        """ Test if the error is raised. """
+    def test_event(self):
+        """ Test if parser treatens ignored values correctly """
         self.allowed_error_count = 1
         self.run_bot()
-        self.assertMessageEqual(0, EXAMPLE_EVENT)
-        self.assertLogMatches("Failed to parse line.")
-        self.assertLogMatches("intelmq.lib.exceptions.InvalidValue: invalid value '-' \(<class 'str'>\) for key 'source.fqdn'")
```

Comment:
I'm not comfortable removing this check. We can use the invalid FQDN value `.` for testing this error behavior.

Reply:
Done